test/e2e/run_test.sh

```diff
@@ -226,6 +226,16 @@ while (( "$#" )); do
       SKIP_BUILD=true
       shift
     ;;
+    --cluster)
+      NAME="$2"
+      if ! kind get clusters | grep -qw "$NAME"; then
```

Comment:
Don't nest `SKIP_SETUP` logic here, use `--cluster` to get cluster name is enough.

Whether to create a cluster follow the previous logic.